.idea.bak/compiler.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you please remove all .idea/* files ?

Reply:
yes the files are removed